### PR TITLE
web/satellite: api key header updated, redundant components removed

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: web/satellite: api key header updated, redundant components removed

Why: due to Satellite UX Enhancements 2.0 api key header was updated. Redundant components like addApiPopup etc were removed

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/project/NewProjectPopup.vue [hunk 1]
```diff
@@ -107,6 +108,8 @@
             this.checkIfsFirstProject();
 
             this.isLoading = false;
+
+            this.$router.push(ROUTES.PROJECT_OVERVIEW.path + '/' + ROUTES.PROJECT_DETAILS.path);
```
Comment (crawter) [inline web/satellite/src/components/project/NewProjectPopup.vue]: remove

Comment (crawter) [reply web/satellite/src/components/project/NewProjectPopup.vue]: add fetchApiKeys method

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 2]
```diff
@@ -2,54 +2,138 @@
...
-        <div v-if="!isEmpty" class="api-keys-container">
-            <div class="api-keys-container__content">
-                <div v-for="apiKey in apiKeyList" v-on:click="toggleSelection(apiKey.id)">
-                    <ApiKeysItem
-                        v-bind:class="[apiKey.isSelected ? 'selected': null]"
-                        :apiKey="apiKey" />
+    <div class="api-keys-area">
+        <h1>Api Key</h1>
+        <div class="api-keys-area__container">
+            <div class="api-keys-area__container__new-api-key" v-if="isNewApiKeyPopupShown">
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: We should consider creating separate component for new api key creation to make ApiKeysArea smaller and moreover simplier

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 3]
```diff
@@ -2,54 +2,138 @@
...
+                    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M15.7071 1.70711C16.0976 1.31658 16.0976 0.683417 15.7071 0.292893C15.3166 -0.0976311 14.6834 -0.0976311 14.2929 0.292893L15.7071 1.70711ZM0.292893 14.2929C-0.0976311 14.6834 -0.0976311 15.3166 0.292893 15.7071C0.683417 16.0976 1.31658 16.0976 1.70711 15.7071L0.292893 14.2929ZM1.70711 0.292893C1.31658 -0.0976311 0.683417 -0.0976311 0.292893 0.292893C-0.0976311 0.683417 -0.0976311 1.31658 0.292893 1.70711L1.70711 0.292893ZM14.2929 15.7071C14.6834 16.0976 15.3166 16.0976 15.7071 15.7071C16.0976 15.3166 16.0976 14.6834 15.7071 14.2929L14.2929 15.7071ZM14.2929 0.292893L0.292893 14.2929L1.70711 15.7071L15.7071 1.70711L14.2929 0.292893ZM0.292893 1.70711L14.2929 15.7071L15.7071 14.2929L1.70711 0.292893L0.292893 1.70711Z" fill="#384B65"/>
+                    </svg>
                 </div>
+                <div class="blur-content"></div>
             </div>
-            <Footer v-if="isSelected"/>
+            <div class="api-keys-area__container__save-api-popup" v-if="isCopyApiKeyPopupShown">
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: Same here

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 4]
```diff
@@ -2,54 +2,138 @@
...
+                                <Button class="button" label="Cancel" width="122px" height="48px" isWhite="true" :onPress="onClearSelection"/>
+                            </div>
+                        </div>
+                    </HeaderComponent>
+                </div>
+                <div v-if="!isEmpty" class="api-keys-items">
+                    <div class="api-keys-items__content">
+                        <div v-for="apiKey in apiKeyList" v-on:click="toggleSelection(apiKey.id)">
+                            <ApiKeysItem
+                                v-bind:class="[apiKey.isSelected ? 'selected': null]"
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: :class='{ selected: apiKey.isSelected }'

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 5]
```diff
@@ -2,54 +2,138 @@
...
-    import AddAPIKeyPopup from '@/components/apiKeys/AddApiKeyPopup.vue';
-    import Footer from '@/components/apiKeys/footerArea/Footer.vue';
-    import { API_KEYS_ACTIONS, APP_STATE_ACTIONS } from '@/utils/constants/actionNames';
-    import { ApiKey } from '../../types/apiKeys';
+    import { API_KEYS_ACTIONS, NOTIFICATION_ACTIONS } from '@/utils/constants/actionNames';
+    import { ApiKey } from '@/types/apiKeys';
+    import Button from '@/components/common/Button.vue';
+    import { RequestResponse } from '@/types/response';
+    import HeaderlessInput from '@/components/common/HeaderlessInput.vue';
+    import VueClipboards from 'vue-clipboards';
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: import disordered

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 6]
```diff
@@ -2,54 +2,138 @@
...
     @Component({
         components: {
             EmptyState,
-            HeaderArea,
+            HeaderComponent,
             ApiKeysItem,
-            AddAPIKeyPopup,
-            Footer,
+            Button,
+            HeaderlessInput
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: missing trailing comma

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 7]
```diff
@@ -2,54 +2,138 @@
...
         },
     })
     export default class ApiKeysArea extends Vue {
         public emptyImage: string = EMPTY_STATE_IMAGES.API_KEY;
+        private name: string = '';
+        private errorMessage: string = '';
+        private key: string = '';
+        private isLoading: boolean = false;
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: data assignment can be simplified like `private isLoading = false;`

File: web/satellite/src/components/apiKeys/ApiKeysArea.vue [hunk 8]
```diff
@@ -2,54 +2,94 @@
...
+            <ApiKeysCopyPopup
+                :isPopupShown="isCopyApiKeyPopupShown"
+                :apiKeySecret="apiKeySecret"
+                @closePopup="closeCopyNewApiKeyPopup"/>
+            <div v-if="!isEmpty" class="api-keys-header">
+                <HeaderComponent ref="headerComponent" placeHolder="API Key">
+                    <div class="header-default-state" v-if="headerState === 0">
+                        <Button class="button" label="+Create API Key" width="180px" height="48px" :onPress="onCreateApiKeyClick"/>
+                    </div>
+                    <div class="header-selected-api-keys" v-if="headerState === 1 && !isDeleteClicked">
```
Comment (crawter) [inline web/satellite/src/components/apiKeys/ApiKeysArea.vue]: !headerState || isDeleteClicked

File: web/satellite/tests/unit/apiKeys/ApiKeysCreationPopup.spec.ts [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+describe('ApiKeysArea async success', () => {
+    let store;
+    let actions;
+    let state;
+    let getters;
+    let apiKey = new ApiKey('testId', 'test', 'test', 'test');
+
+    beforeEach(() => {
+        actions = {
```
Comment (crawter) [inline web/satellite/tests/unit/apiKeys/ApiKeysCreationPopup.spec.ts]: remove

**Pull request merged**